src/main/java/com/airbnb/billow/ARNGenerator.java

```diff
@@ -33,7 +34,10 @@ private static String loadAccountNumber(AmazonIdentityManagementClient iamClient
         String userARN = null;
 
         try {
-            GetUserResult getUserResult = iamClient.getUser();
+            final String USER_NAME = "MikeCurtis";
```

Comment:
Please, this is a fully configurable open source project. Can you use the config file?

https://github.com/airbnb/billow/blob/master/src/main/resources/reference.conf for the defaults/documentation
https://github.com/airbnb/billow/blob/master/src/main/java/com/airbnb/billow/Main.java#L49 for where the AWS-specific config is first constructed (then passed around).